<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWindow</class>
 <widget class="QDialog" name="OptionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_general">
      <attribute name="title">
       <string>General</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_fonts">
      <attribute name="title">
       <string>Fonts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_cardname">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cardname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_cardname">
         <property name="currentFont">
          <font>
           <family>Times New Roman</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_abilities">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Abilities</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_abilities">
         <property name="currentFont">
          <font>
           <family>Adobe Fangsong Std R</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_flavor">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Flavor text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_flavor">
         <property name="currentFont">
          <font>
           <family>Times New Roman</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_stats">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Stats</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_stats">
         <property name="currentFont">
          <font>
           <family>Georgia</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cardtype">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cardtype</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_cardtype">
         <property name="currentFont">
          <font>
           <family>Times New Roman</family>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_voidcost">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Void Cost Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="font_voidcost">
         <property name="currentFont">
          <font>
           <family>Georgia</family>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
